libs/openFrameworks/graphics/ofTrueTypeFont.cpp

```diff
@@ -600,10 +600,10 @@ bool ofTrueTypeFont::load(string _filename, int _fontSize, bool _bAntiAliased, b
 			// 1-bit format, hella funky
 			// here we unpack it:
 			unsigned char *src =  bitmap.buffer;
-			for(unsigned int j=0; j <bitmap.rows;j++) {
```

Comment:
also we have width and height as local variables a some lines before in that function so we can do:

```
for(int j=0; j <height;j++) {
```